dmoj/graders/bridged.py

```diff
@@ -54,14 +59,21 @@ def _launch_process(self, case):
         os.close(submission_stdout_pipe)
 
     def _interact_with_process(self, case, result, input):
-        output = case.output_data()
+        judge_output = case.output_data()
         self._interactor_time_limit = (self.handler_data.preprocessing_time or 0) + self.problem.time_limit
         self._interactor_memory_limit = self.handler_data.memory_limit or env['generator_memory_limit']
+        args = self.handler_data.args or contrib_modules[self.contrib_type].ContribModule.get_interactor_args_string()
```

Comment:
`s/args/args_format_string/g` everywhere, because we already use `args` to mean a *list* when it comes to generators so this would be overloading the same key (albeit with a different parent).